cdv/cmds/sim_utils.py

```diff
@@ -280,10 +294,25 @@ async def async_config_wizard(
 
     await generate_plots(config, root_path, fingerprint)
     # final messages
-    print(f"\nFarming & Prefarm reward address: {config['simulator']['farming_address']}\n")
+    farming_address = config["simulator"]["farming_address"]
+    print(f"\nFarming & Prefarm reward address: {farming_address}\n")
     print("Configuration Wizard Complete.")
     print("Starting Simulator now...\n\n")
     await start_async(root_path, ("simulator",), False)
+    # now we create make sure the simulator has a genesis block
+    print("Please wait, generating genesis block.")
+    while True:
+        try:
+            num_blocks = await execute_with_simulator(None, root_path, get_current_height, False)
+        except ClientConnectorError:
+            await asyncio.sleep(0.25)
+        else:
+            if num_blocks == 0:
+                await execute_with_simulator(None, root_path, farm_blocks, False, 1, True, farming_address)

```

Comment:
Should we just return here since we are guaranteed to be height 1

Reply:
The create command can also be used to modify an existing simulator, so I think this should stay.